app/src/main/res/layout/fragment_add_new_login.xml

```diff
@@ -0,0 +1,253 @@
+        android:contentDescription="@string/saved_login_clear_username"
+        app:tint="@color/saved_login_clear_edit_text_tint"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="@id/inputLayoutUsername"
+        app:srcCompat="@drawable/ic_clear" />
+
+    <TextView
+        android:id="@+id/passwordHeader"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:gravity="center_vertical"
+        android:paddingStart="3dp"
+        android:paddingEnd="0dp"
+        android:text="@string/preferences_passwords_saved_logins_password"
+        android:textColor="?primaryText"
+        android:textSize="12sp"
+        android:layout_marginTop="10dp"
+        android:letterSpacing="0.05"
+        app:fontFamily="@font/metropolis_semibold"
+        app:layout_constraintBottom_toTopOf="@id/inputLayoutPassword"
+        app:layout_constraintEnd_toStartOf="@id/revealPasswordButton"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/inputLayoutUsername"
+        app:layout_constraintVertical_chainStyle="packed" />
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/inputLayoutPassword"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:colorControlActivated="?primaryText"
+        android:colorControlHighlight="?primaryText"
+        android:contentDescription="@string/saved_login_password_description"
+        android:paddingBottom="11dp"
+        android:textColor="?primaryText"
+        app:hintEnabled="false"
+        app:layout_constraintEnd_toEndOf="@id/passwordHeader"
+        app:layout_constraintStart_toStartOf="@id/passwordHeader"
+        app:layout_constraintTop_toBottomOf="@id/passwordHeader"
+        app:layout_constraintVertical_chainStyle="packed">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/passwordText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:clickable="true"
+            android:colorControlActivated="?primaryText"
+            android:colorControlHighlight="?primaryText"
+            android:cursorVisible="true"
+            android:ellipsize="end"
+            android:focusable="true"
+            android:fontFamily="sans-serif"
+            android:inputType="textNoSuggestions"
+            android:letterSpacing="0.01"
+            android:lineSpacingExtra="8sp"
+            android:maxLines="1"
+            android:singleLine="true"
+            android:textColor="?primaryText"
+            android:textCursorDrawable="@null"
+            android:textSize="16sp"
+            android:textStyle="normal"
+            app:backgroundTint="?primaryText"
+            tools:ignore="Autofill" />
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <ImageButton
```

Comment:
I think we can remove this button here. I don't think we need it if we're creating a new item